make/upgrade

```diff
@@ -8,16 +8,24 @@ set -o errexit -o nounset
 
 NAMESPACE="cf"
 UAA_NAMESPACE="uaa"
-UAA_CA_CERT=internal-ca-cert
 
 GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "${GIT_ROOT}"
 
-# Determine the kube revision of the chart controlling the specified namespace.
-release_version() { helm list --date --reverse --max 1 --namespace "$1" | awk '{ print $2 }' | tail -n 1 ; }
+get_uaa_secret_name() {
```

Comment:
I think this function should take an argument for `INTERNAL_CA_CERT` and not have it hard-coded.

Reply:
Done with commit 405a6bdb262d41482dfb7d8e5045cf3f406aa183